src/libui_sdl/main.cpp

```diff
@@ -1328,7 +1331,7 @@ void OnDropFile(uiWindow* window, char* file, void* blarg)
     char* ext = &file[strlen(file)-3];
     int prevstatus = EmuRunning;
 
-    if (!strcasecmp(ext, "nds") || !strcasecmp(ext, "srl"))
+    if (!strcasecmp(ext, "nds") || !strcasecmp(ext, "srl") || !strcasecmp(ext, "zip") || !strcasecmp(ext, ".7z"))
```

Comment:
It may be a good idea to use `CheckArchiveExtensions` here to simplify adding further archive formats in the future.

Reply:
Done